packages/fluentui/docs/src/examples/components/Accordion/Rtl/AccordionExample.rtl.tsx

```diff
@@ -3,13 +3,11 @@ import { Accordion } from '@fluentui/react-northstar';
 
 const panels = [
   {
-    key: 'first',
-    title: 'مرحبا العالم',
+    title: { key: 'content-1', content: 'مرحبا العالم' },
     content: 'مرحبا العالم',
   },
   {
-    key: 'second',
-    title: 'مرحبا المريخ',
+    title: { key: 'content-2', content: 'مرحبا المريخ' },
```

Comment:
cool! can you also make all the keys unique in the Accordion example? Esclusive and Exclusive and Expanded have the same keys (oups my fault) but since you created this PR ... :D

Reply:
also probably it is better to use `rtl-content-2` just to be super sure we will not use the same keys on another example. Same as below for alert

Reply:
Done